feat(curriculum): add binary search workshop (#60727)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-binary-search/68416aab85dd00a7852cdeb6.md

---
id: 68416aab85dd00a7852cdeb6
title: Step 15
challengeType: 20
dashedName: step-15
---

# --description--

> You can see that the last call prints `This shows because the value is farther than the middle value to the right or left`. That's not what you want, so instead of returning that message, return an empty list and the message `Value not found`.

Reviewer: ```suggestion
You can see that the last call prints an empty list. To further specify that the value is not in the searched list, update the `return` statement and return an empty list and the message `Value not found`.
```

# --hints--

> You should replace the returned empty list with `[], 'Value not found'`.

Reviewer: ```suggestion
You should update the `return` statement to return `[], 'Value not found'`.
```
